app/models/stage.rb

```diff
@@ -42,6 +42,14 @@ def single_sided?
     double_elim? || single_sided_swiss? || single_elim?
   end
 
+  def is_swiss?
```

Comment:
it might be nice to add a quick test for these in spec/models/stage_spec.rb, even though this is dead simple.